curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-algorithmic-thinking-by-building-a-dice-game/64a1a59b5a5d6c013d95306d.md

```diff
@@ -0,0 +1,302 @@
+---
+id: 64a1a59b5a5d6c013d95306d
+title: Step 18
+challengeType: 0
+dashedName: step-18
+---
+
+# --description--
+
+You need a function to reset the game, so create a function named `resetGame`.
+
+# --hints--
+
+You should create a function named `resetGame`
```

Comment:
`resetGame` is defined as an arrow function in [the final UI](https://github.com/JoyShaheb/CurriculumExpansion/blob/6041c265812423a54cb20cd3b307e5b8634e401a/javascript-algorithms-and-data-structures/learn-intermediate-algorithmic-thinking-by-building-a-dice-game/index.js#L223).

Reply:
added option for both normal and es6 arrow function